### ISSUE TITLE
first few notes are not exported due to missing directory, error disappears when folder structure is present

### ISSUE THREAD

**Issue by msrlab** [akosbalasko/yarle#428]
I use the command line to execute the script 

`npx -p yarle-evernote-to-md@latest yarle --configFile <path_to_your_file e.g. ./config.json>`

and a customized config file for exporting to logseq.

Notes are exported nicely and a folder structure is created, but the first few notes are skipped.
It seems to be related to the creation of the folder structure because this is the error in the conversion.log:

```
Failed to convert note: testnote
{"errno":-2,"syscall":"scandir","code":"ENOENT","path":"/Users/mypath/testing/assets"}
Conversion finished at Sat Dec 03 2022 06:42:49 GMT+0100 (Central European Standard Time)
Note "testnote" converted successfully in 0.13 seconds.
```

If I delete all exported notes and assets/files but preserve the folder structure, everything works fine onthe second pass of executing the script

This behaviori reproducible for every export I tested.
npx version 9.1.3
MacOs Big Sur 11.6.7 (20G630)

**Comment by akosbalasko** [comment]
Hi @msrlab ,
thank you for raising this issue. I'll try to resolve it snd release a fixed version soon.

**Comment by msrlab** [comment]
Great, and thanks a lot for this super helpful app!

**Comment by akosbalasko** [comment]
it's fix was released in version 5.0.3, so I close this issue.
